src/Controller/PageAdminController.php

```diff
@@ -256,22 +268,18 @@ public function composeAction(Request $request): Response
      */
     public function composeContainerShowAction(Request $request): Response
     {
-        $blockAdmin = $this->container->get('sonata.page.admin.block');
-        \assert($blockAdmin instanceof BlockAdmin);
-
-        if (false === $blockAdmin->isGranted('LIST')) {
+        //TODO Can I use this directly from symfony controller?
+        if (false === $this->isGranted('LIST')) {
```

Comment:
nope, this should remain as using blockAdmin, it generates the permission different. And IMO we should change for $blockAdmin->checkAccess('list') (but I don't see any relation with your bug report tbh, so it should be done in a separate PR).

Reply:
yeah it's not related with my issue, I'm just trying to make my test simple!

Reply:
If you provide failing test when you changed src, we can't know if the test if failing because of the bug or because of your changes. So src code should not be changed.